obspy/io/resp/nrl.py

```diff
@@ -207,6 +210,10 @@ def parse_entire_library(self):
         """
         Parse the entire NRL
         """
+        if not self._local:
+            msg = ("Parsing the entire NRL library via http can take a *very* "
+                   "long time.")
+            warnings.warn(msg)
```

Comment:
I would just not allow this. It takes a really long time and I don't think the IRIS servers would enjoy it.